portal/views/staff.py

```diff
@@ -147,12 +152,15 @@ def staff_index():
     ).join(UserOrganization).filter(
         and_(UserOrganization.user_id == User.id,
              UserOrganization.organization_id.in_(org_list)))
-    staff_list = staff_list.union(org_staff).all()
 
-    # only show test users to admins
-    if not user.has_role(ROLE.ADMIN.value):
-        staff_list = [s for s in staff_list if not s.has_role(ROLE.TEST.value)]
+    include_test_roles = request.form.get('include_test_roles')
```

Comment:
Does this option need to be passed by POST?

It's usually easier to work with (debug/bookmark/reproduce ) GET requests and querystring parameters

Reply:
thanks, ivan, fixed